cmake/configure/configure_1_lapack.cmake

```diff
@@ -34,8 +34,6 @@ MACRO(FEATURE_LAPACK_FIND_EXTERNAL var)
     SET(CMAKE_REQUIRED_LIBRARIES
       ${DEAL_II_LINKER_FLAGS_SAVED} ${LAPACK_LINKER_FLAGS} ${LAPACK_LIBRARIES}
       )
-    # Push -pthread as well:
-    ENABLE_IF_SUPPORTED(CMAKE_REQUIRED_FLAGS "-pthread")
```

Comment:
I suppose you are removing this here because we now unconditionally do this?

Reply:
Yes.